kratos/tests/test_levelset_convection.py

```diff
@@ -150,8 +144,8 @@ def test_levelset_convection_BFECC(self):
         # gid_output.ExecuteFinalizeSolutionStep()
         # gid_output.ExecuteFinalize()
 
-        self.assertAlmostEqual(max_distance, 1.0617777301844604)
-        self.assertAlmostEqual(min_distance, -0.061745786561321375)
+        self.assertAlmostEqual(max_distance, 1.0634680107706003)
+        self.assertAlmostEqual(min_distance, -0.06361967738862996)

```

Comment:
The value change is due to the gradient calculation at each time step right?

Reply:
Yes. It slightly changes the result.